README.md

```diff
@@ -15,4 +16,28 @@ markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
 
+## Definition of little $o$ 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+## Definition of Big $O$
+$f(n)\in O(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) \le c g(n)$
+
+## Proof 
+- If we start with $f(n)\in o(g(n))$ by the deffiniton provided we know that for any positive constant $c$ there exists a point called 
+  $n_0$ beyond which is always $\ f(n) < c \cdot g(n)$
+- If we were to choose another constant $k$ that is greater then 0.
+- We can then use the new constant for the definition of big $O$ to show that $f(n)\in O(g(n))$
+- Since we know that $\ \forall n\ge n_0: f(n) < c g(n)$
+- We can see that $\ f(n) \le k g(n)$ using the new constant $k$.
```

Comment:
Still not following you here. How do you know that $k$ is a valid constant here?